appchain-anchor/src/appchain_messages.rs

```diff
@@ -98,8 +98,12 @@ impl IndexedAndClearable for u32 {
         ()
     }
     //
-    fn clear_extra_storage(&mut self) {
-        ()
+    fn clear_extra_storage(&mut self) -> MultiTxsOperationProcessingResult {
```

Comment:
Why is this function named `clear_extra_storage`?

Reply:
It is for clear extra storage data besides the data of type `T` itself in `LookupArray`. If some collection fields are used in type `T`, the implementation of `T` should implement this function to clear all extra storage keys they used. For example, we may use `LookupMap` or `UnorderedSet` in type `T`, then we should implement this function to clear all storage keys used in these collection types.

Reply:
Ok, I get it.